### PR TITLE
turn off fmr default metric for embeddings.py

### PR DESCRIPTION
**Pull request by Natyren**
and delete fmr_values in val_sop.yaml

For now, we don't push you to follow any predefined schema of issue, but ensure you've already read our contribution guide: https://open-metric-learning.readthedocs.io/en/latest/from_readme/contributing.html.

### CODE REVIEW

**Comment by AlekseySh** [comment]
@Natyren Oh, we already had an empty argument for the FNMR metric. I thought it was not empty. Thus, we don't need to change the code at all.

What we really need to do instead is to remove the metric from ``val_sop.yaml`` (already done in this PR) and from ``val_inshop.yaml`` (https://github.com/OML-Team/open-metric-learning/blob/main/examples/inshop/configs/val_inshop.yaml#L33)

**Comment by Natyren** [comment]
@AlekseySh Change val_inshop.yaml, sorry for miscommunication on first commit

File: oml/metrics/embeddings.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(
         cmc_top_k: Tuple[int, ...] = (5,),
         precision_top_k: Tuple[int, ...] = (5,),
         map_top_k: Tuple[int, ...] = (5,),
-        fmr_vals: Tuple[float, ...] = tuple(),
+        fmr_vals: Tuple[float, ...] = None,
```
Comment (AlekseySh) [inline oml/metrics/embeddings.py:70]: we don't to change this line, it was already empty
so, roll back this particular line please

**Review by AlekseySh — CHANGES_REQUESTED** [review]

File: oml/metrics/embeddings.py (hunk 1 above)
Comment (Natyren) [reply oml/metrics/embeddings.py]: done

**Review by AlekseySh — APPROVED** [review]
LGFM, @Natyren thank you for your first contribution to OML!

**Pull request merged**